cli/make-dereferenced-schemas.js

```diff
@@ -7,18 +7,22 @@ const schemaRefParser = require(`json-schema-ref-parser`);
 
 const fixtureSchema = require(`../schemas/fixture.json`);
 const manufacturersSchema = require(`../schemas/manufacturers.json`);
+const definitionsSchema = require(`../schemas/definitions.json`);
 
 dereference(fixtureSchema, path.join(__dirname, `../schemas/dereferenced/fixture.json`));
 dereference(manufacturersSchema, path.join(__dirname, `../schemas/dereferenced/manufacturers.json`));
+dereference(definitionsSchema, path.join(__dirname, `../schemas/dereferenced/definitions.json`));
 
 /**
  * Resolve all "$ref" properties in the given schema and save the resolved version.
  * @param {!object} schema The raw JSON schema data.
  * @param {!string} newFilename The absolute path to the dereferenced schema.
  */
 function dereference(schema, newFilename) {
+  process.chdir(path.join(__dirname, `../schemas/`));
+
   schemaRefParser.dereference(schema).then(dereferencedSchema => {
```

Comment:
`x => { return doStuff(...); }` should always be replaced by `x => doStuff(...)`

Reply:
Done.